csharp/ql/lib/semmle/code/csharp/dataflow/SSA.qll

```diff
@@ -34,11 +52,10 @@ module Ssa {
       or
       // Local variable declaration without initializer
       not exists(result.getTargetAccess()) and
-      this =
-        any(SourceVariables::LocalScopeSourceVariable v |
-          result.getTarget() = v.getAssignable() and
-          result.getEnclosingCallable() = v.getEnclosingCallable()
-        )
+      exists(LocalScopeVariable v, Callable c |
+        assignableDefinitionLocalScopeVariable(result, v, c) and
+        localScopeSourceVariable(this, v, c, _)
```

Comment:
Did you need to force the join order for performance reasons?

Reply:
Yes.